xml/XSLT/panelfile-2-9-6.xsl

```diff
@@ -1,23 +1,10 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<!-- $Id$ -->
 
-<!-- Stylesheet to convert a JMRI panel file into an HTML page -->
+<!-- Stylesheet to convert a JMRI panel file into an HTML page              -->
+<!-- Used by default when the panel file is displayed in a web browser      -->
+<!-- This version corresponds to the 2.9.6 schema update                    -->
 
-<!-- Used by default when the panel file is displayed in a web browser-->
-
-<!-- Thus version corresponds to the 2.9.6 schema update -->
-
-<!-- This file is part of JMRI.  Copyright 2007-2011.                            -->
-<!--                                                                        -->
-<!-- JMRI is free software; you can redistribute it and/or modify it under  -->
-<!-- the terms of version 2 of the GNU General Public License as published  -->
-<!-- by the Free Software Foundation. See the "COPYING" file for a copy     -->
-<!-- of this license.                                                       -->
-<!--                                                                        -->
-<!-- JMRI is distributed in the hope that it will be useful, but WITHOUT    -->
-<!-- ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or  -->
-<!-- FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License  -->
-<!-- for more details.                                                      -->
+<!-- This file is part of JMRI.  Copyright 2007-2011-2016.                       -->
```

Comment:
The copyright should either be continuous (look at the files history) of a set of comma separated years or ranges (something like "2007-2011, 2016")